sympy/combinatorics/perm_groups.py

```diff
@@ -3178,25 +3176,6 @@ def is_group(self):
         So we see that g[1] and g[3] are equivalent to their inverse while
         g[7] == ~g[2].
         """
-        # identity present
-        I = Permutation(size=self.degree)
-        for g in self:
-            if g == I:
-                break
-        else:
-            return False
-
-        # associativity already holds: a*(b*c) == (a*b)*c for permutations
-
-        # inverse of each is present
-        if not all(self.has(~a) for a in self):
-            return False
-
-        # closure
-        for a in self:
-            for b in self:
-                if not self.has(a*b):
-                    return False
         return True
```

Comment:
I think it could even be possible to define ``is_group`` as a class attribute: ``PermutationGroup.is_group = True``.